### DISCUSSION TITLE
Lists suggestions

### DISCUSSION THREAD

**Discussion by bananaturtlesandwich** [community/community#8759] [Other Feature Feedback, Questions, & Ideas]
This is a really nice idea! But I think a some other things could make it even better!
When adding repos to lists can it be made so the repository isn't starred so repos don't clutter the starred list too much
Can sub-lists be added so navigation is easier?

**Comment by brianlovin** [comment]
Thanks for the suggestions! I've passed both along to our team to help us prioritize the next iterations 😊
